website/content/en/docs/concepts/disruption.md

```diff
@@ -307,8 +324,18 @@ Duration and Schedule must be defined together. When omitted, the budget is alwa
 
 ### Pod-Level Controls
 
-You can block Karpenter from voluntarily choosing to disrupt certain pods by setting the `karpenter.sh/do-not-disrupt: "true"` annotation on the pod. This is useful for pods that you want to run from start to finish without disruption. By opting pods out of this disruption, you are telling Karpenter that it should not voluntarily remove a node containing this pod.
-
+You can block Karpenter from voluntarily disrupting and draining pods by adding the `karpenter.sh/do-not-disrupt: "true"` annotation to the pod.
+You can treat this annotation as a single-pod, permanently blocking PDB.
+This has the following consequences:
+- Nodes with `do-not-disrupt` pods will be excluded from [Consolidation]({{<ref "#consolidation" >}}), and conditionally excluded from [Drift]({{<ref "#drift" >}}).
+  - If the Node's owning NodeClaim has a [`terminationGracePeriod`]({{<ref "#terminationgraceperiod" >}}) configured, it will still be eligible for disruption via drift.
```

Comment:
Can we reword to capture that it was the TGP set on the NodePool for which the node launched?

Reply:
I think we can probably update the doc sections for expiration and TGP to emphasize that they are driftable fields persisted to the NodeClaim. I'd rather state that there than need to clarify everywhere we discuss either of those fields.